openstack/hermes/templates/api-deployment.yaml

```diff
@@ -19,55 +19,60 @@ spec:
-            - -f
-            - /etc/hermes/hermes.conf
-          env:
-            - name: HERMES_DEBUG
-              value: '{{.Values.hermes.debug}}'
-            - name: HERMES_INSECURE
-              value: '{{.Values.hermes.insecure}}' # SSL certificates are hard :(
-            - name: HERMES_ES_USERNAME
-              valueFrom:
-                secretKeyRef:
-                  name: hermes-es-api
-                  key: HERMES_ES_USERNAME
-            - name: HERMES_ES_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: hermes-es-api
-                  key: HERMES_ES_PASSWORD
-          volumeMounts:
-            - mountPath: /etc/hermes
-              name: config
-          resources:
-            requests:
-              memory: {{required ".Values.resources_requests_memory_api is missing" .Values.resources_requests_memory_api}}
-              cpu: {{required ".Values.resources_requests_memory_api is missing" .Values.resources_requests_cpu_api}}
-          # livenessProbe:
-          #   #exec: 
-          #   #  command:
-          #   #    - /bin/false
-          #   #  initialDelaySeconds: 21600
-          #   #  periodSeconds: 21600 # restart api every 6 hours
-          #   httpGet:
-          #     path: /v1/
-          #     port: 80
-          #   timeoutSeconds: 10
-          #   periodSeconds: 60
-          #   initialDelaySeconds: 60
-          #   readinessProbe:
-          #     httpGet:
-          #       path: /v1/
-          #       port: 80
-          #     timeoutSeconds: 5
-          #     periodSeconds: 5
+      - name: api
+        image: {{required ".Values.hermes.image is missing" .Values.hermes.image}}:{{.Values.hermes.image_tag | replace "string:" ""}}
+        imagePullPolicy: {{required ".Values.hermes.image_pull_policy is missing" .Values.hermes.image_pull_policy}}
+        env:
+        - name: HERMES_DEBUG
+          value: '{{.Values.hermes.debug}}'
+        - name: HERMES_INSECURE
+          value: '{{.Values.hermes.insecure}}' # SSL certificates are hard :(
+        - name: HERMES_ES_URL
+          value: "https://{{.Values.hermes_elasticsearch_host}}.{{.Values.global.region}}.{{.Values.global.tld}}:{{.Values.hermes_elasticsearch_port}}"
+        - name: HERMES_ES_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: hermes-es-api
+              key: HERMES_ES_USERNAME
+        - name: HERMES_ES_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: hermes-es-api
+              key: HERMES_ES_PASSWORD
+        - name: HERMES_OS_AUTH_URL
+          value: "{{.Values.hermes.auth_url}}"
+        - name: HERMES_OS_USERNAME
+          value: "{{.Values.hermes.username}}"
+        - name: HERMES_OS_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: hermes-api
+              key: HERMES_OS_PASSWORD
+        - name: HERMES_OS_AUTH_URL
+          value: "{{.Values.hermes.auth_url}}"
```

Comment:
NIT: These URLs could also be templated as such: https://github.com/sapcc/helm-charts/blob/40b53f86d5f7f555c946a52d536177de9728abd3/openstack/nova/templates/etc/_nova.conf.tpl#L99